package.json

```diff
@@ -19,12 +19,16 @@
     "type": "git",
     "url": "git://github.com/goodeggs/json-fetch.git"
   },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "always-auth": true
+  },
   "homepage": "https://github.com/goodeggs/json-fetch",
   "bugs": "https://github.com/goodeggs/json-fetch/issues",
   "dependencies": {
     "babel-runtime": "6.26.0",
     "isomorphic-fetch": "2.2.1",
-    "lodash.pick": "4.4.0",
```

Comment:
I don't feel strongly about this, but the idea here was to avoid pulling in a pretty large dependency in an open-source module (which might be used in browsers, where bundle size matters). It doesn't matter much for us internally because we're using lodash anyway, but it might for others. 🤷‍♂️